deploy/firesim

```diff
@@ -104,56 +85,57 @@ def runworkload(runtime_conf):
     """ do runworkload. """
     runtime_conf.run_workload()
 
-def buildafi(globalbuildconf):
+def buildafi(build_config_file):
     """ Starting from local Chisel, build an AFI for all of the specified
     hardware configs. """
 
-    auto_create_bucket(globalbuildconf.s3_bucketname)
-
-    #check to see email notifications can be subscribed
-    get_snsname_arn()
+    # forced to build locally
+    for build_config in build_config_file.builds_list:
+        execute(replace_rtl, build_config, hosts=['localhost'])
+        execute(build_driver, build_config, hosts=['localhost'])
 
-    def terminate_instances_handler(sig, frame):
-        """ Handler that prompts to terminate build instances if you press ctrl-c. """
+    def release_build_farm_hosts_handler(sig, frame):
+        """ Handler that prompts to release build farm hosts if you press ctrl-c. """
         rootLogger.info("You pressed ctrl-c, so builds have been killed.")
         userconfirm = input("Do you also want to terminate your build instances? Type 'yes' to do so.\n")
         if userconfirm == "yes":
-            globalbuildconf.terminate_all_build_instances()
-            rootLogger.info("Instances terminated. Please confirm in your AWS Management Console.")
+            build_config_file.release_build_farm_hosts()
+            rootLogger.info("Build farm hosts released.")
```

Comment:
I've always thought it was shrewd of @sagark to suggest that you double-check via the Build Farm Provider (AWS Management Console up until now).  Perhaps we should think about expanding this suggestion rather than just dropping it?

Reply:
This should already have a message printed in the `release_build_farm_host` function that mentions checking the AWS Management Console. https://github.com/firesim/firesim/blob/417ebe654d8ee6c71c18cd473bd0e4dc2632352b/deploy/buildtools/buildfarmhostdispatcher.py#L301